### PR TITLE
Feature/11 style intro slide

### LINKED ISSUE #11: Style intro slide

### PR DESCRIPTION
**Pull request by bellangerq**
Basic intro slide styling, considering it's an MVP.

Closes #11.

### CODE REVIEW

File: src/styles/utility.css [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .flow > * + * {
-  margin-top: 2rem;
+  margin-top: var(--spacing-l, var(--flow-spacing));
```
Comment (hissalht) [inline src/styles/utility.css:2]: Did you mean to do this ?
```suggestion
  margin-top: var(--flow-spacing, var(--spacing-l));
```

File: src/styles/variables.css [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  :root {
+    --font-size-m: 1rem;
+    --font-size-l: 1.75rem;
+    --font-size-xl: 2rem;
+    --font-size-xxl: 2.5rem;
+
+    --spacing-m: 0.5rem;
+    --spacing-l: 1rem;
+    --spacing-xl: 2rem;
+    --spacing-xxl: 4rem;
```
Comment (hissalht) [inline src/styles/variables.css:35]: Not sure how this will interact with customizable theme in the future 🤔

File: src/styles/utility.css (hunk 1 above)
Comment (bellangerq) [reply src/styles/utility.css:2]: Yes!

File: src/styles/variables.css (hunk 2 above)
Comment (bellangerq) [reply src/styles/variables.css:35]: Yeah for now I didn't take this into account (how much the thing is customizable: every single variable or only colors/fonts for example).

**Pull request merged**